chia/rpc/harvester_rpc_api.py

```diff
@@ -35,13 +36,17 @@ async def _state_changed(self, change: str, change_data: Dict[str, Any] = None)
 
         if change == "farming_info":
             payloads.append(create_payload_dict("farming_info", change_data, self.service_name, "metrics"))
+            payloads.append(create_payload_dict("farming_info", change_data, self.service_name, "wallet_ui"))
 
         if change == "add_connection":
             payloads.append(create_payload_dict("add_connection", change_data, self.service_name, "metrics"))
 
         if change == "close_connection":
             payloads.append(create_payload_dict("close_connection", change_data, self.service_name, "metrics"))
 
+        if change == "harvesting_mode_changed":
```

Comment:
Currently only "local" harvester's harvesting mode (CPU/GPU) is interested by the GUI. Besides, communication between farmer <-> harvester is done via Peer protocol (protocols defined under `chia/protocols`) and rarely use RPC API from my quick look. I think updating peer protocol should be avoided as long as possible because it is part of consensus.